udf/bytesmask_365_days.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TEMP FUNCTION
+  udf_bytesmask_365_days() AS (
+    CONCAT(
+        b'\x1F\xFF\xFF\xFF',
```

Comment:
Looks like we could make this less verbose as `RPAD(b'\x1F', 46, b'\xFF')` or perhaps read legibly `CONCAT(b'\x1F', REPEAT(b'\xFF', 45))`.

Reply:
Funny, we just reached the same conclusion independently!

Reply:
I created an issue to follow up on ways we can make more maintainable bit logic: https://github.com/mozilla/bigquery-etl/issues/578